docs/pages/integration-test/masonry.js

```diff
@@ -64,12 +101,17 @@ export default function TestPage(): Node {
           externalCache={booleanize(externalCache)}
           finiteLength={booleanize(finiteLength)}
           flexible={booleanize(flexible)}
-          initialItems={generateExampleItems({ name: 'InitialPin' })}
+          initialItems={
+            realisticPinHeights
+              ? generateRealisticExampleItems({ name: 'InitialPin', pinHeightsSample })
+              : generateExampleItems({ name: 'InitialPin' })
```

Comment:
How come the older example is still supported? Would we not always use realistic pin heights